lib/chat_api/reporting.ex

```diff
@@ -40,6 +40,26 @@ defmodule ChatApi.Reporting do
     |> Repo.all()
   end
 
+  def average_first_replied_time(account_id, filters \\ %{}) do
```

Comment:
let's add typespecs to these functions, it should help with warnings/readability/IDE experience improvements 👍 

e.g.
```elixir
@spec average_first_replied_time(binary(), map()) :: float()
```

also I wonder if we should make it clearer that the output of this is in `seconds` (vs `ms` or something else)... maybe we could name it something like `average_seconds_to_first_reply` or `average_time_to_first_reply_in_seconds`? not sure 🤷